libs/utils/fedsHelpers.js

```diff
@@ -0,0 +1,29 @@
+import { getConfig } from './utils.js';
+
+let federatedContentRoot;
+/* eslint-disable import/prefer-default-export */
+export const getFederatedContentRoot = () => {
+  const defaultAllowedOrigins = [
+    'https://www.adobe.com',
+    'https://business.adobe.com',
+    'https://milo.adobe.com',
+    'https://news.adobe.com',
```

Comment:
I think you can re-add blog here, [Chris' comment](https://github.com/adobecom/milo/pull/2531#discussion_r1669375066) resolved around blog not having the georouting feature.
From an akamai perspective, it's set up to support federal features though to improve loading times.

Good that we moved this into a separate file, since it already caused confusion 😁